src/main/java/com/gistpetition/api/DataLoader.java

```diff
@@ -75,30 +77,48 @@ public void loadData() {
-        Petition petition11 = savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal);
-        Petition petition12 = savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal);
-        savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal);
-        savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal);
-        savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal);
-        List<Petition> petitions = Arrays.asList(petition1, petition2, petition3, petition4, petition5, petition6, petition7, petition8, petition9, petition10, petition11, petition12);
+        List<Petition> petitions = List.of(
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("국민 청원에 글을 써버렸다.", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal),
+                savePetition("2차방역지원금 지금 자영업분들 농락하시나요?", normal)
+        );
+        savePetition("Temp 2차방역지원금 지금 자영업분들 농락하시나요?", normal);
+        savePetition("Temp 2차방역지원금 지금 자영업분들 농락하시나요?", normal);
+        savePetition("Temp 2차방역지원금 지금 자영업분들 농락하시나요?", normal);
 
-        List<Integer> numOfAgree = List.of(26, 13, 16, 21, 6, 9, 8, 5, 5, 21, 17, 16);
+        Random random = new Random();
         for (int i = 0; i < petitions.size(); i++) {
             Petition petition = petitions.get(i);
-            for (int j = 0; j < numOfAgree.get(i); j++) {
+            int agreeCount = random.nextInt(alphabetUsers.size() - REQUIRED_AGREEMENT) + REQUIRED_AGREEMENT;
+            for (int j = 0; j < agreeCount; j++) {
                 User user = alphabetUsers.get(j);
                 petitionService.agree(AGREEMENT_REQUEST, petition.getId(), user.getId());
             }
+            if (i < 3) {
```

Comment:
```suggestion
            if(i<tempPetitionCount) {
```
 와 같이 기준을 따로 빼 두면 어떨까요? 이 부분은 나중에 리팩토링 하면 좋을 것 같아요!

Reply:
은기와 비슷한 피드백이네요! 반영했습니다!